lib/flipper/api/v1/actions/groups_gate.rb

```diff
@@ -0,0 +1,49 @@
+require 'flipper/api/action'
+require 'flipper/api/v1/decorators/feature'
+
+module Flipper
+  module Api
+    module V1
+      module Actions
+        class GroupsGate < Api::Action
+          route %r{api/v1/features/[^/]*/groups/?\Z}
+
+          def post
+            ensure_valid_params
+            feature = flipper[feature_name]
+            feature.enable_group(group_name)
+            decorated_feature = Decorators::Feature.new(feature)
+            json_response(decorated_feature.as_json, 200)
+          end
+
+          def delete
+            ensure_valid_params
+            feature = flipper[feature_name]
+            feature.disable_group(group_name)
+            decorated_feature = Decorators::Feature.new(feature)
+            json_response(decorated_feature.as_json, 200)
+          end
+
+          private
+
+          def ensure_valid_params
+            json_response({ code: 100, message: 'Feature not found.', more_info: '' }, 404) unless feature_names.include?(feature_name)
```

Comment:
Any particular reason for 100 and 200 as the codes? I ask because I would perhaps lean toward just starting with 1 and 2. Having a 3 digit number feels to close to an http status perhaps.